main.c

```diff
@@ -299,13 +299,23 @@ libusb_device_handle * LIBUSB_CALL open_device_with_vid(
 					directory = INSTALL_PREFIX "/share/rpiboot/mass-storage-gadget64/";
 					use_bootfiles = 1;
 					snprintf(bootfiles_path, sizeof(bootfiles_path),"%s/%s", directory, "bootfiles.bin");
-					printf("2712: Directory not specified using default %s\n", directory);
+					printf("2712: Directory not specified trying default %s\n", directory);
```

Comment:
A separator of some kind makes the message easier to parse, e.g. "...not specified - using default"